rest/data/admin.go

```diff
@@ -56,7 +56,11 @@ func SetEvergreenSettings(ctx context.Context, changes *restModel.APIAdminSettin
 		// get the property name and find its value within the settings struct
 		propName := changesReflect.Type().Field(i).Name
 		changedVal := changesReflect.FieldByName(propName)
-		if changedVal.IsNil() {
+		if changedVal.IsZero() {
```

Comment:
IsZero checks IsNil for structs/points, and zero values for other fields (it's effectively it but better in this scenario)